colon-app/src/main/java/pl/edu/agh/to/app/controller/HomeController.java

```diff
@@ -1,17 +1,27 @@
 package pl.edu.agh.to.app.controller;
```

Comment:
I would be grateful if you look at new changes (Spring Boot) and make this file consistent :)